x/ibc/20-transfer/keeper/relay.go

```diff
@@ -209,7 +209,7 @@ func (k Keeper) refundPacketAmount(ctx sdk.Context, packet channel.Packet, data
 	}
 
 	// check the denom prefix
-	prefix := types.GetDenomPrefix(packet.GetSourcePort(), packet.GetSourceChannel())
+	prefix := types.GetDenomPrefix(packet.GetDestPort(), packet.GetDestChannel())
```

Comment:
Refunds are always processed on sending chain so SRC and DST order is same as the way it is in `SendTransfer`

Reply:
can you write that as a comment on the code?

Reply:
This is a mistake in the spec, sorry - fixed in https://github.com/cosmos/ics/commit/a819bb56f28a2b2678ab1ec02f417bfb8c93565b